lib/libsm/include/sm3.h

```diff
@@ -0,0 +1,39 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2019 Huawei Technologies Co., Ltd
+ */
+/**
+ * \file sm3.h
+ * thanks to Xyssl
+ * SM3 standards:http://www.oscca.gov.cn/News/201012/News_1199.htm
```

Comment:
check url. Maye find a better one from  the "Definitive standards" section of [wikipedia](https://en.wikipedia.org/wiki/SM3_(hash_function)).

Reply:
Indeed it's broken :-/
I'll remove it. A valid link is in the commit description.